src/instructlab/sdg/default_flows.py

```diff
@@ -226,9 +227,10 @@ def get_flow(self) -> list:
                 "block_config": {
                     "block_name": "filter_relevancy",
                     "filter_column": "score",
-                    "filter_value": 2.0,
+                    "filter_value": 2,
+                    "valid_values": [0, 1, 2],
```

Comment:
Same here - if we're only accepting `score=2` then why is `valid_values` needed?

Reply:
Refer to above reply, same case